pkg/resolver/search.go

```diff
@@ -186,7 +188,10 @@ func (s *SearchResult) resolveUids() {
 }
 func (s *SearchResult) resolveItems() ([]map[string]interface{}, error) {
 	items := []map[string]interface{}{}
+	timer := prometheus.NewTimer(metric.DBQueryDuration.WithLabelValues("what-do-we-put"))
+	klog.Infof("xxxxxxxxxxxxxxxx [%s] ", s.query)
 	rows, err := s.pool.Query(context.Background(), s.query, s.params...)
+	timer.ObserveDuration()
```

Comment:
Actually we need to add `defer` to run this when exiting the function because the query results will be "streaming" as we process it.
```suggestion
	defer timer.ObserveDuration()
```

Reply:
done. Thanks. I missed this!